lib/internal/process/report.js

```diff
@@ -105,6 +107,12 @@ const report = {
 
     nr.setReportOnUncaughtException(trigger);
   },
+  get excludeEnv() {
```

Comment:
Would you mind adding a setter like `set reportOnSignal` as well?

Reply:
Done 
fed0381